UITests/AuthenticationTests.swift

```diff
@@ -70,13 +70,15 @@ class AuthenticationTests: KIFTestCase {
 
     fileprivate func loadAuthPage() {
         EarlGrey.select(elementWithMatcher: grey_accessibilityID("url")).perform(grey_tap())
-        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_typeText("\(webRoot!)/auth.html\n"))
+        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_replaceText("\(webRoot!)/auth.html"))
+        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_typeText("\n"))
```

Comment:
So what happens occasionally on ipad is that after certain key combo, it cannot find the key element and fails to type.  Way to work around this was to use replaceText() instead of typing it.